node/src/main/scala/coop/rchain/node/configuration/Configuration.scala

```diff
@@ -36,9 +36,8 @@ object Configuration {
 
   private val DefaultPort                       = 40400
   private val DefaultGrpcPortExternal           = 40401
-  private val DefaultHttPort                    = 40402
-  private val DefaultMetricsPort                = 40403
-  private val DefaultGrpcPortInternal           = 40404
+  private val DefaultGrpcPortInternal           = 40402
+  private val DefaultHttPort                    = 40403
```

Comment:
@rabbitonweb Please run integration tests on your local machine before merging to `dev`.

Reply:
```
===========================================================
=================TEST SUMMARY RESULTS======================
PASS: peer1.rchain.coop: Peers count correct in node logs.
PASS: peer0.rchain.coop: Peers count correct in node logs.
PASS: peer1.rchain.coop: Rholang evaluation of files performed correctly.
PASS: peer0.rchain.coop: Rholang evaluation of files performed correctly.
PASS: peer0.rchain.coop: REPL loader success!
PASS: peer1.rchain.coop: Proposal of blocks for deployed contracts worked.
PASS: peer0.rchain.coop: Proposal of blocks for deployed contracts worked.
PASS: bootstrap.rchain.coop: Proposal of blocks for deployed contracts worked.
PASS: peer1.rchain.coop: No errors defined by "ERROR" in logs.
PASS: peer0.rchain.coop: No errors defined by "ERROR" in logs.
PASS: peer1.rchain.coop: No text of "RuntimeException" in logs.
PASS: peer0.rchain.coop: No text of "RuntimeException" in logs.
PASS ALL: All tests successfully passed
===========================================================
===========================================================
```